ua/org.eclipse.help.webapp/m/index.js

```diff
@@ -99,20 +105,20 @@
                          var scopeName = decodeHtml(match[1]);
                          if (scopeName.substring(0, 1) == '\u200B') {
                              if (scopeName.length < 3) {
-                                 searchScope.l = scopeName.length;
+                                 searchScope.level = scopeName.length;
                                  break;
                              } else {
-                                 searchScope.l = 4;
-                                 searchScope.t = scopeName.length - 3;
+                                 searchScope.level = SEARCH_SCOPE_LEVEL_CUSTOM;
```

Comment:
SEARCH_SCOPE_LEVEL_CUSTOM is 3 now (was 4). Expected?

Reply:
Yes. 3 was reserved for a future use that never happened. Now it is a constant that is used as an enum value. I tested it manually by setting these scope level constants to random numbers to be sure not to miss something.